full.yml

```diff
@@ -603,6 +604,30 @@
       tags: proxy
   tags: drone
 
+## Drone docker runner
+
+- name: Deploy drone docker runner in docker
+  hosts: all
+  become: true
+  gather_facts: false
+  vars:
+    enable_db: false
+    service_name: drone-docker-runner
+    docker_image: "{{ drone_docker_runner_docker_image }}"
+    docker_command: |
+      /usr/bin/docker run --rm --name "{{ service_name }}" \
+      -v /var/run/docker.sock:/var/run/docker.sock \
+      -p "{{ drone_docker_runner_internal_web_port }}:3000" \
+      -e "DRONE_RPC_PROTO=https" \
+      -e "DRONE_RPC_HOST={{ drone_domain }}" \
+      -e "DRONE_RPC_SECRET={{ drone_rpc_scret }}" \
+      -e "DRONE_RUNNER_CAPACITY=2" \
+      -e "DRONE_RUNNER_NAME={{ drone_runner_name }}" \
+      "{{ docker_image }}"
+  roles:
+    - role: generic_docker_systemd
+  tags: drone
+
```

Comment:
If docker runner is deployed in docker it won't be able to execute things outside of it right? For example `molecule test` using Vagrant.

Reply:
On the next PR I will add docker exec that can run molecule test. Sometimes its better to run pipelines with docker. The best idea is to have the docker runner and the exec runner.

Reply:
Great :-)